docs/pages/user-docs/subpages.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Subpages
+parent: User Guide
+nav_order: 640
+---
+# :diamond_shape_with_a_dot_inside: Subpages
+## :trident: Organigram 
+
+This subpage offers you a visualization of our organizational structure. By scrolling down the page you find different work units which can be expanded by clicking on the arrow far to the right. 
+## :door:  Rooms 
+When you open the room subpage you can directly open our internal **booking tool** and check for available facilities as in Outlook. If this doesn't work, the function is disabled by your company
+
+To find a specific room, you need to navigate either via the column at the left side or by clicking the buttons displayed in the middle. 
+![screenshot of room subpage ](media/rooms_demo.PNG)
+
+At the next deeper level you find **contact information** for the whole location you selected and the option to open it in **Google Maps**. 
+![screenshot of room subpage with selected city ](media/rooms_demo2.PNG)
+If you go further you can select a **floor** and than a specific **room**.
+
+## :open_file_folder: Profile menu
+When you open the profile menu you can navigate to further subpages. 
+
+![open menu option at the very right, listing subpages "settings", "information", "release notes" and "report a bug"](media/profilemenu_demo.PNG)  
+
+| Subpage                                    | Content                                                                                                                                                                                                                                                                                                                                                                |
+| ------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **My Profile :woman:**                     | This is were you can view the stored **information** about you. They will be displayed to other users when they open your profile. There's also the possibility to upload a **profile picture** as well as changing or deleting the current one. A profile picture facilitates recognizing you in the search. If the displayed information is wrong please contact XY. |
+| **Settings :gear:**                        | You can switch the Phonebook from **English** to **German** with a single click.To personalize your Phonebook you can also choose between four **design options**: a dark or light background and blue or magenta writings.                                                                                                                                            |
```

Comment:
don't describe the designs in a to detail way. the problem here is, if we add more designs we must always update the guide. that's not practical.